ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -634,6 +638,11 @@ private void processGetBlobResponse(GetRequestInfo getRequestInfo, GetResponse g
           if (getError == ServerErrorCode.No_Error) {
             handleBody(getResponse.getInputStream());
             chunkOperationTracker.onResponse(getRequestInfo.replicaId, true);
+            if (RouterUtils.isRemoteReplica(routerConfig, getRequestInfo.replicaId)) {
+              logger.trace("Cross colo request successful for remote replica in ",
```

Comment:
1. This is what we have been counting for the coordinator. It helps in knowing how well cross-colo requests are helping. One should always compare the total and successful together (which makes an error count unnecessary).
2. Strictly not necessary as the class name is added to the log message.